casepro/backend/junebug.py

```diff
@@ -357,7 +366,22 @@ def push_outgoing(self, org, outgoing, as_broadcast=False):
         :param as_broadcast: whether outgoing messages differ only by recipient and so can be sent as single broadcast
         """
         for message in outgoing:
-            self.message_sender.send_message(message)
+            self.send_message(message)
+
+    def send_message(self, outgoing):
+        message = outgoing.reply_to
```

Comment:
It might be a good idea to add a default channel field on the contact, so we default back to the last channel used by the contact. Not necessary but would be nice.